src/utils.py

```diff
@@ -12,7 +13,15 @@
 ]
 
 
-PREFIX = "deeporigin://"
+@dataclass
+class PREFIXES:
```

Comment:
defined all constants prefixes here. this allows me to refer to any prefix easily using `PREFIXES.DO` e.g.